game/src/core/utils/components/path/SimpleIPath.java

```diff
@@ -29,4 +29,9 @@ public String pathString() {
   public int priority() {
     return 0;
   }
+
+  @Override
+  public String toString() {
```

Comment:
wo kommt das denn grad her? hat da was mit dem aufteilen zu tun?

Reply:
Hatte ich eingebaut, damit man in den Error-Messages wenigstens mal ne Info hat welcher Pfad nicht gefunden wird, anstatt nur die Klasse mit ObjektId.

Reply:
ah, ok.